libgadget/treewalk.c

```diff
@@ -876,7 +889,10 @@ int treewalk_visit_ngbiter(TreeWalkQueryBase * I,
     }
 
     lv->Ninteractions += ninteractions;
-    lv->Nnodesinlist += inode;
+    if(lv->mode == 1) {
+        lv->Nnodesinlist += inode;
+        lv->Nlist += 1;
```

Comment:
Is Nlist the number of nodelists used for this ghost particle, or used for this particle (sum of all ghost particles of the real particle)?

Reply:
The latter